packages/examples/src/bakeoff/reddit-qa/ai-jsx.tsx

```diff
@@ -35,11 +35,11 @@ const docs = await Promise.all(
 const corpus = new LocalCorpus(staticLoader(docs), defaultChunker);
 await corpus.startLoading();
 
-function ShowDoc({ doc }: { doc: Document<RedditMetadata> }) {
+function ShowDoc({ doc }: { doc: ScoredChunk }) {
```

Comment:
I think it's somewhat important to show that the metadata type still lines up.

```suggestion
function ShowDoc({ doc }: { doc: ScoredChunk<RedditMetadata> }) {
```

I think that means you could continue using the title from the metadata as a first choice.

Reply:
I wasn't able to get this to work properly with TS checks, it complained that `RedditMetadata` was not `Jsonifiable`. Do you want to take another run at this?

Reply:
Hmm... I had initially defined RedditMetadata as an interface that extends JsonObject to avoid exactly that problem. The "MustBeJsonObject" alternative that exists now was Nick's suggestion.  @NickHeiner Can you help with this?

(I also think it's fine to submit as is and I can clean it up later.)

Reply:
OK, SGTM.